app/lib/features/news/pages/add_news_page.dart

```diff
@@ -51,16 +51,13 @@ class _AddNewsState extends ConsumerState<AddNewsPage> {
           ? Colors.transparent
           : selectedNewsPost?.backgroundColor,
       actions: [
-        Visibility(
-          visible: selectedNewsPost?.type == NewsSlideType.text,
-          child: IconButton(
-            onPressed: () {
-              ref
-                  .read(newsStateProvider.notifier)
-                  .changeTextSlideBackgroundColor();
-            },
-            icon: const Icon(Atlas.color),
-          ),
+        IconButton(
+          onPressed: () {
+            ref
+                .read(newsStateProvider.notifier)
+                .changeTextSlideBackgroundColor();
+          },
+          icon: const Icon(Atlas.color),
```

Comment:
inkwell and long-press to allow for a custom color selector? Well, future issue, I guess...

Reply:
Yes, In future we can provider option to select custom color. But currently implement with fixed set of bg color look good to me and it is making things simple.